formulaire.html

```diff
@@ -0,0 +1,106 @@
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="formImage">
+        <img src="img\image-11.png" alt="">
+    </div>
+    <!--the first div of formulaire-->
+    <div class="formulaire">
+        <h2 class="titles">Python developper</h2>
+
+        <span class="formcat">
+                    <!--categorie of formation-->
+                <a href="">Informatique</a>
+                </span>
+        <!--description of formation&-->
+        <p>
+            Cette formation a pour objectif de permettre de maîtriser la syntaxe essentielle de Python. Etre capable de créer et maintenir des applications complètes et complexes en Python
+        </p>
+        <!--finishing the description-->
+
+
+
+        <!--container of a propos -->
+        <div class="apropos">
+            <div class="smallContainer">
+                <img src="img\global.svg" alt="">
+                <span>
+                <h3>100% Online</h3>    
+                <p>
+                Start instantly and learn in you own schedule
+                </p>
+                </span>
+            </div>
+
+            <div class="smallContainer">
+                <img src="img\timer.svg" alt="">
+                <span>
+                <h3>Flexible terms</h3>    
+                <p>
+                Reset deadlines in accordance to your schedule
+                </p>
+                </span>
+            </div>
+            <!--fin de small container -->
+            <div class="smallContainer">
+                <img src="img\chart.svg" alt="">
+                <span>
+                <h3>Flexible terms</h3>    
+                <p>
+                You will learn from the beginning
+                </p>
+                </span>
+            </div>
+            <!--fin de small container -->
+        </div>
+    </div>
+    <h2 class="titles">Postuler a cette formation</h2>
+    <form class="register" action="#" method="post">
+        <fieldset>
+            <legend>Nom et Prenom</legend>
+            <input type="text" name="name" id="">
+        </fieldset>
+        <fieldset>
+            <legend>Date de naissance</legend>
+            <input type="text" name="name" id="">
+        </fieldset>
+        <fieldset>
+            <legend>Email</legend>
+            <input type="text" name="name" id="">
+        </fieldset>
+        <fieldset>
+            <legend>Tel</legend>
+            <input type="text" name="name" id="">
+        </fieldset>
+        <fieldset>
+            <legend>Sex</legend>
+            <input type="text" name="name" id="">
+        </fieldset>
+        <fieldset>
+            <legend>Niveau detudes</legend>
+            <input type="text" name="name" id="">
+        </fieldset>
+        <fieldset>
+            <legend>Message</legend>
+            <textarea name="message" id="" cols="30" rows="10">
+
+            </textarea>
+        </fieldset>
+        <input type="submit" value="S'inscrire">
+    </form>
+    <footer>
```

Comment:
include your footer.html

Reply:
do you mean to code <?php include"footer.html" ?> !!